### ISSUE TITLE
VnV Report: Nonfunctional quality 2

### ISSUE THREAD

**Issue by MolyMikhail** [4G06-CAPSTONE-2025/Reading4All#658]
Many other aspects are touched on but there is not much data given to support them. They say pass/fail but without much data to back up those claims. I would have liked to see more data for accessibility in particular. The tool discussed in section 9 was very interesting, I'd love to see the data.
